src/zenml/enums.py

```diff
@@ -38,11 +38,15 @@ class StepRunInputArtifactType(StrEnum):
     MANUAL = "manual"  # manually loaded via `zenml.load_artifact()`
 
 
-class StepRunOutputArtifactType(StrEnum):
-    """All possible types of a step run output artifact."""
+class ArtifactSaveType(StrEnum):
+    """All possible method types of how artifact versions can be saved."""
 
     DEFAULT = "default"  # output of the current step
```

Comment:
Not part of your changes, but what do you think of renaming this to something like `STEP_OUTPUT`? If I was a user and read `save_type=default` on an `ArtifactVersionResponse`, I would have no clue what this is referring to I think